### PR TITLE
Feature/play for ether frontend

### PR DESCRIPTION
**Pull request by dennis-westphal**
## Description

Added possibility to play for ether to the frontend
## Related Issue
#92 Display won ether
#88 Add ether display to join game page
#87 Specify ether in create game page
## How Has This Been Tested?

Started multiple games and played against the other browsers ;-)
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist contributor:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
## Checklist reviewer:
- [ ] The code has been peer-reviewed by @YourNameHere

### CODE REVIEW

File: app/javascript/gamesFactory.js [hunk 1]
```diff
@@ -89,25 +92,33 @@ angular.module('dappChess').factory('games', function (navigation, accounts, $ro
...
       }
     }
 
-    game.ended = contractGameObject.ended;
+    if (contractGameObject.ended) {
+      game.ended = true;
+    }
+    else {
+      game.ended = false;
+    }
```
Comment (kyroy) [inline app/javascript/gamesFactory.js]: This may be changed to `game.ended = contractGameObject.ended`

**Comment by kyroy** [comment]
> You can tell a friend to directly join the game with the game id
> 0x02b8362bac8f0d4cf6f1faf2c88bbcba6302b2c168458e6dda3507267dc9597b

I think we should remove this, if we remove the ability to join a game by id.

**Comment by dennis-westphal** [comment]
Done

File: app/javascript/initializeGame.js [hunk 2]
```diff
@@ -15,9 +16,12 @@ angular.module('dappChess').controller('InitializeGameCtrl',
       $rootScope.$broadcast('message', 'Your game is being created, please wait a moment...',
                             'loading', 'startgame');
       try {
-        
+
         Chess.initGame($scope.username, $scope.startcolor === 'white',
-                     { from: $scope.selectedAccount });
+         {
+           from: $scope.selectedAccount,
+           value: web3.toWei($scope.etherbet.replace(',', '.'), 'ether')
```
Comment (kyroy) [inline app/javascript/initializeGame.js:23]: `$scope.etherbet` should be checked. It is a `type="text"` field and the user can enter arbitrary things ;)

**Comment by kyroy** [comment]
Looks great. Contains changes from #91 which are not completed (and corrent) yet, but they fit the purpose of this feature. They have to be reviewed in another pull request ;)

**Pull request merged**